backend/devices/bitbox02/keystore.go

```diff
@@ -70,6 +69,20 @@ func (keystore *keystore) SupportsAccount(coin coin.Coin, meta interface{}) bool
 	}
 }
 
+// SupportsAccount implements keystore.Keystore.
+func (keystore *keystore) SupportsAccount(coin coin.Coin, meta interface{}) bool {
+	if !keystore.SupportsCoin(coin) {
+		return false
+	}
+	switch coin.(type) {
+	case *btc.Coin:
+		scriptType := meta.(signing.ScriptType)
```

Comment:
Isn't this going to panic if passed a different type? Wouldn't you want:

```go
typ, ok := meta.(signing.ScriptType)
```

If it's always `signing.ScriptType`, definitely consder changing the `meta` arg to a script type.

Reply:
It is only a ScriptType for BTC/LTC. I wish we had Rust enums (union types).

Reply:
If it were me, would've never guessed it and needed to `grep SupportsAccount`, hoping to find all the references across the codebase. I guess we'll leave it for another day.